### PR TITLE
Improve info output

### PR DESCRIPTION
**Pull request by matthiasclasen**
Emit more information, and format it in a better way.

### CODE REVIEW

**Comment by alexlarsson** [comment]
This breaks the tests, mainly because they expect "flatpak info --show-commit app" to only show the commit.

That seems like a useful option, to avoid parsing the generic info output, although i like the new formating as the default.

**Comment by matthiasclasen** [comment]
Yeah, I wasn't quite sure which way to go here - we could leave the individual --show- options for 'machine parsable' output, and just display the nice formatted output for when now --show options are given ?

**Comment by alexlarsson** [comment]
@matthiasclasen yeah, i think that makes sense

File: doc/flatpak-info.xml [hunk 1]
```diff
@@ -44,6 +44,12 @@
             Show info about and installed application and/or runtime.
         </para>
         <para>
+            By default, the output if formatted in a friendly format.
```
Comment (AsciiWolf) [inline doc/flatpak-info.xml]: Typo: if formatted -> is formatted

**Pull request merged**